sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncContainer.java

```diff
@@ -154,7 +154,7 @@ public String getId() {
     public Mono<CosmosAsyncContainerResponse> replace(
         CosmosContainerProperties containerProperties,
         CosmosContainerRequestOptions options) {
-        ModelBridgeInternal.validateResource(containerProperties);
+        ModelBridgeInternal.validateResource(ModelBridgeInternal.getResourceFromResourceWrapper(containerProperties));
```

Comment:
We are validating these resources in RxDocumentClientImpl anyway, so we could skip these altogether? same for database too. Minor change, need not worry about this in this PR.